.prettierrc.json

```diff
@@ -1,8 +1,8 @@
 {
-	"printWidth": 100,
-	"semi": true,
-	"singleQuote": true,
-	"trailingComma": "all",
-	"useTabs": true,
-	"tabWidth": 2
+  "printWidth": 100,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "useTabs": true,
```

Comment:
The irony of seeing the prettier configs file being reformatted without considering the rules specified therein (tabs)
😂

Reply:
for clarity, comment not intended as anything else but to point out I find this funny.